tests/graphics/ellipsization.cpp

```diff
@@ -101,8 +101,9 @@ TEST_CASE_METHOD(EllipsizationTestCase, "Ellipsization::NormalCase", "[ellipsiza
                     WX_ASSERT_MESSAGE
                     (
                      (
-                        "Test #(%u,%u.%u): \"%s\" -> \"%s\"; width=%dpx > %dpx",
+                        "Test #(%u,%u.%u): %s\n\"%s\" -> \"%s\"; width=%dpx > %dpx",
```

Comment:
I think a more useful assert message - taking into account the other related PR - might be as follows:
```
WX_ASSERT_MESSAGE
(
 (
    "Test #(%u,%u.%u): \n%s\n%s\n\"%s\" -> \n\"%s\"; width=%dpx > %dpx",
    s, f, m,
    getenv("GDK_DPI_SCALE"),
    dc.GetFont().GetNativeFontInfoUserDesc(),
    str,
    displayed,
    width,
    widthsToTest[w]
 ),
 width <= widthsToTest[w]
);

```

Reply:
I intentionally omitted this because it just doesn't seem to make sense to show the value of this env var here. I.e. I don't see myself doing it without writing a lengthy comment or at least a commit message and I don't think it's worth it.

It would be more useful to output the information about scaling/DPI on the startup of the GUI tests (or maybe as a side effect of some test?) instead.

Reply:
Okay, I didn't realise that I had made a commit with the extended information in it. 

Agreed, I now realise there is very limited benefit to including the additional information in this message.